.github/workflows/How_To_Release.md

```diff
@@ -0,0 +1,48 @@
+# How to publish a new HyTEST release
+
+## Essential steps
+* Make `dev` branch look like you want.  
+  * Merge various feature branches in to `dev` with PRs (or not... manually also works; PR preferred)
+  * Direct commits to `dev` are also OK (but discouraged)
+  * edit the `environment_set_up/manifest.txt` to reflect what you want in the bootstrap archive (see
+    below: "What's In the Archive").
+* Create a PR to merge `dev` into `main`
+  * Use github **Labels** to identify how you want the Semantic Version to Increment:
```

Comment:
Could you include some info on what types of changes would constitute each level of increment change?

Reply:
Done.